### ISSUE TITLE
mamba broke my conda installation

### ISSUE THREAD

**Issue by isuruf** [QuantStack/mamba#50]
```
## Package Plan ##

  environment location: /home/isuru/miniconda3

  added / updated specs:
    - pyopencl=2019.1


The following packages will be downloaded:

    package                    |            build
    ---------------------------|-----------------
    cython-0.29.12             |   py37he1b5a44_0         2.2 MB  conda-forge
    more-itertools-7.1.0       |             py_0          46 KB  conda-forge
    pyopencl-2019.1            |   py37h9de70de_0         649 KB  conda-forge
    pyrsistent-0.15.3          |   py37h516909a_0          89 KB  conda-forge
    pytest-5.0.1               |           py37_0         347 KB  conda-forge
    ruamel.yaml-0.15.98        |   py37h516909a_0         325 KB  conda-forge
    ------------------------------------------------------------
                                           Total:         3.6 MB

The following NEW packages will be INSTALLED:

  pyopencl           conda-forge/linux-64::pyopencl-2019.1-py37h9de70de_0
  wheel              conda-forge/linux-64::wheel-0.33.4-py37_0

The following packages will be UPDATED:

  cython                             0.29.11-py36he1b5a44_0 --> 0.29.12-py37he1b5a44_0
  more-itertools     conda-forge/linux-64::more-itertools-~ --> conda-forge/noarch::more-itertools-7.1.0-py_0
  pillow                               6.0.0-py36he7afcd5_0 --> 6.1.0-py37he7afcd5_0
  pyrsistent                          0.15.2-py36h516909a_0 --> 0.15.3-py37h516909a_0
  pytest                                       5.0.0-py36_0 --> 5.0.1-py37_0
  python                                3.6.7-h381d211_1004 --> 3.7.3-h33d41f4_1
  readline                                7.0-hf8c457e_1001 --> 8.0-hf8c457e_0
  ruamel.yaml                        0.15.96-py36h516909a_0 --> 0.15.98-py37h516909a_0

The following packages will be DOWNGRADED:

  conda-build                                 3.18.6-py36_0 --> 3.17.8-py37_1
```

Updated python from 3.6 to 3.7 without updating conda

**Comment by wolfv** [comment]
hi @isuruf thanks for the bug report. indeed, i think i need to teach mamba to keep environments consistent. need to look into that.

**Comment by wolfv** [comment]
ok i am pretty sure i found and fixed the issue in #51

**Comment by wolfv** [comment]
this should be fixed now! a "sub-issue" is #52 where dependencies are not as greedily updated as they are with conda.

**Comment by wolfv** [comment]
btw I released 0.0.8 for linux and OS X just now. Windows needs another small fix (adding back a std::string_view polyfill).
